packages/next/src/views/Document/renderDocumentSlots.tsx

```diff
@@ -40,23 +40,22 @@ export const renderDocumentSlots: (args: {
     })
   }
 
-  const descriptionFromConfig =
-    collectionConfig?.admin?.description || globalConfig?.admin?.description
+  const globalDescriptionFromConfig = globalConfig?.admin?.description
 
-  const staticDescription: StaticDescription =
-    typeof descriptionFromConfig === 'function'
-      ? descriptionFromConfig({ t: req.i18n.t })
-      : descriptionFromConfig
+  const staticGlobalDescription: StaticDescription =
+    typeof globalDescriptionFromConfig === 'function'
+      ? globalDescriptionFromConfig({ t: req.i18n.t })
+      : globalDescriptionFromConfig
 
   const CustomDescription =
-    collectionConfig?.admin?.components?.Description ||
+    collectionConfig?.admin?.components?.edit?.Description ||
```

Comment:
In a project where you have a `collection.admin.components.Description`, you'd no longer see that in the edit view it.

It seems to me that we need the CustomDescription to be:

```ts
const CustomDescription =
    collectionConfig?.admin?.components?.edit?.Description ||
    collectionConfig?.admin?.components?.Description ||
    globalConfig?.admin?.components?.elements?.Description
```

Without this existing projects will be forced to add the `collectionConfig.admin.components.edit.Description` with the same value the `Description` or suddently an update to Payload will stop showing the `Description` they once had in edit view. This could be considered a breaking change.

Reply:
I hear you. I've added the fallback and reverted `staticGlobalDescription` to how it was before to accommodate for list view descriptions. I've also added docs and caught a minor issue in the collections component table that prevented it from rendering correctly.

Thanks for taking the time to review.